schedulezoomtalks.py

```diff
@@ -0,0 +1,99 @@
+import os
+import github
+import requests
+from researchseminarsdotorg import *
+from requests import get, post, put
+from io import StringIO
+from ruamel.yaml import YAML
+from common import *
+
+TALKS_FILE = "speakers_corner_talks.yml"
+
+def schedule_zoom_talk(talk) -> string:
+
+    # Form the talk registration body
+    request_body =
+    {
+      "topic": "Speakers\' corner talk by %s"%(talk.get("name")),
+      "type": 2, # Scheduled meeting
+      "start_time": talk.get("time"), #2020-03-31T17:00:00
+      "timezone": "UTC",
+      "duration": 60, # 90 minutes
+      "schedule_for": "vsf@virtualscienceforum.org",  # Zoom user ID or Zoom email address
+      "agenda":talk.get("title"),
+      "password": "",
+
+      # Meeting settings
+      "settings": {
+        "host_video": True,
+        "participant_video": True,
+        "cn_meeting": False,  # Host the meeting in China?
+        "in_meeting": False,  # Host the meeting in India?
+        "join_before_host": False,
+        "mute_upon_entry": True,
+        "watermark": False,  # Add a watermark when screensharing?
+        "use_pmi": False, # Don't use the Personal Meeting ID, but generate one
+        "approval_type": 0, # Automatically approve
+        "audio": "both",
+        "auto_recording": "cloud",
+        "enforce_login": False,
+        "alternative_hosts": "",
+        "registrants_email_notification": True,
+        "contact_email": "vsf@virtualscienceforum.org",
+      }
```

Comment:
Need to add custom registration questions

Reply:
Not sure how we should go about this; the schema doesn't seem to include it, though there are fields that are being referred to that are not in the schema either: https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate

Reply:
Try `get`ting a meeting with questions. I remember seeing those, so let's hope the docs are just incomplete.

Reply:
For reference: https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingregistrantquestionupdate
Worth trying just adding this part to the meeting creation body. Will test

Reply:
@akhmerov I added a questions bit to the body, including a bunch of standard ones (that seem unnecessary from your LRC output). May be worth testing the API with that body via the bottom part of https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate